package.json

```diff
@@ -13,8 +13,8 @@
     "packages/*"
   ],
   "scripts": {
-    "build": "yarn ts-bridge --project tsconfig.build.json --verbose && yarn workspace @metamask/design-system-twrnc-preset build && yarn workspace @metamask/design-system-react-native build",
-    "build:clean": "rimraf dist '**/*.tsbuildinfo' && yarn build",
+    "build": "yarn workspaces foreach --all --topological-dev run build",
+    "build:clean": "rimraf 'packages/*/dist' '**/*.tsbuildinfo' && yarn build",
```

Comment:
Refactor build scripts to use Yarn workspaces' native build ordering

Replaces TypeScript project references with `yarn workspaces foreach --all --topological-dev`, which:
1. Automatically determines the correct build sequence based on workspace dependencies
2. Executes each package's complete `build` script rather than just TypeScript compilation
3. Ensures additional build steps (like design tokens build:css) are properly included

This approach simplifies maintenance by eliminating manual build order management and guarantees all build steps across packages are executed in the correct dependency order.

Also updates clean script to properly target all workspace package dist folders.